packages/vats/src/vat-provisioning.js

```diff
@@ -50,6 +51,7 @@ export function buildRootObject(_vatPowers) {
         // Update the notifier when the chainBundle resolves.
         const { notifier, updater } = makeNotifierKit();
         chainBundle.then(clientHome => {
+          // @ts-expect-error IterationObserver not callable
           updater(harden({ clientHome, clientAddress: address }));
```

Comment:
split in multiple lines and cast as appropriate?

Reply:
`updater` itself has the error,
> This expression is not callable.
>  Type 'IterationObserver<any>' has no call signatures.